### PR TITLE
Update test.md

### PR DESCRIPTION
**Pull request by VincentTano**
Added a comment about adding AntShares and AntShares.VM references through Nuget and minor edits

### CODE REVIEW

**Comment by canesin** [comment]
Hey, awesome work.. I just changed the branch to vicentt (I know, I missed a "n" .. thats ok). Our work-flow is a little different to accommodate the squash & merge of ANS and to keep track of contributions to include in our accounting.

1 - We fork on CoZ, them for on personal .. 
2 - Work and PR to CoZ for review.. this pr should not be to master, but to a feature branch as master is supposed to be in sync with ANS.
3 - The original PR author to CoZ submit a PR after review from CoZ to ANS.
4 - After ANS accept we sync master again.

You are now at 2! :+1:  .. just added a comment.

File: en-us/sc/test.md [hunk 1]
```diff
@@ -31,13 +31,15 @@ namespace AntShares.SmartContract
 }
 ```
 
-After compiling, generate the contract's `Test1.avm` file. We can create unit test items and test `Test1.avm`.
+After compiling, generate the contract's `Test1.avm` file. We can create a unit test project and test `Test1.avm`.
 
-First create a C# console project with Visual Studio. Then add a reference to `AntShares.dll` and `AntShares.VM.dll`.
+First create a C# Console App (.Net Framework) project with Visual Studio, with .NET Framework 4.6.2. Then add a reference to `AntShares.dll` and `AntShares.VM.dll`.
```
Comment (canesin) [inline en-us/sc/test.md]: Does it need to be .NET 4.6.2 ? Seams restrictive, can't I use other versions and even .NET core ?

Comment (VincentTano) [inline en-us/sc/test.md]: Thanks for the correction. The latest Chinese version of test.md said 4.6.2 or later so I put that in. I've only been able to test .net 4.6.1 in addition with success. I tried .NET core initially but ran into a bunch of errors when compiling, but might be due to my lack of understanding of Visual C#. Gotta see if anyone else went through this already using .NET core

Comment (canesin) [inline en-us/sc/test.md]: Just remove the version and for me you are good to go.

**Comment by canesin** [comment]
Good to go. I will give you collaboration and you submit the PR to official repo by yourself.

**Comment by VincentTano** [comment]
Sorry I'm not familiar with submitting PRs to a separate repo. Is it
base fork: Antshares/docs base: master ... head fork: CityOfZion/docs compare: vicentt ? Thanks
pr

**Comment by canesin** [comment]
yes, correct.

**Pull request merged**